### PR TITLE
BaSP-TG-60: get employees with redux

### PR DESCRIPTION
**Pull request by Maxi-SR**
- Create constants to get employees
- Create actions to get employees
- Create thunk to get employees
- Create employees reducer
- Connect list component with store to get employees
Only get the list of employees, I removed add, projects, edit and delete button to make it more consistent the PR.
Since the PR of projects is not yet merged, I do not show the projects of an employee.

### CODE REVIEW

File: src/Components/Employees/index.js [hunk 1]
```diff
@@ -1,142 +1,35 @@
-import { useParams, useHistory } from 'react-router-dom';
-import { useEffect, useState } from 'react';
+import { useEffect } from 'react';
 import styles from './employees.module.css';
-import Button from '../Shared/Button/index';
-import Modal from '../Shared/Modal/Modal';
 import Table from '../Shared/Table';
+import { useSelector } from 'react-redux';
+import { useDispatch } from 'react-redux';
```
Comment (Juanoauil) [inline src/Components/Employees/index.js:5]: you can write `import { useSelector, useDispach} from 'react-redux';` together in the same line

File: src/redux/employees/reducer.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import {
+  GET_EMPLOYEES_PENDING,
+  GET_EMPLOYEES_REJECTED,
+  GET_EMPLOYEES_FULFILLED
+} from './constants';
+
+const INITIAL_STATE = {
+  employees: [],
```
Comment (Juanoauil) [inline src/redux/employees/reducer.js:8]: Why do you declare this variable if you don't use it?

File: src/redux/employees/reducer.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      return {
+        ...state,
+        isLoading: true
+      };
+    case GET_EMPLOYEES_FULFILLED:
+      return {
+        ...state,
+        isLoading: false,
+        error: '',
+        list: action.payload
```
Comment (Juanoauil) [inline src/redux/employees/reducer.js:25]: Please save in employees

File: src/Components/Employees/index.js (hunk 1 above)
Comment (Maxi-SR) [reply src/Components/Employees/index.js:5]: sorry, done.

File: src/redux/employees/reducer.js (hunk 2 above)
Comment (Maxi-SR) [reply src/redux/employees/reducer.js:8]: done

File: src/redux/employees/reducer.js (hunk 3 above)
Comment (Maxi-SR) [reply src/redux/employees/reducer.js:25]: done

Comment (Maxi-SR) [reply src/redux/employees/reducer.js:25]: I used list instead of employees

**Review by mara666 — APPROVED** [review]
Well done Maxi!

**Review by Juanoauil — APPROVED** [review]

**Pull request merged**